src/vshaxe/Main.hx

```diff
@@ -80,6 +79,7 @@ class Main {
 		function maybeStartServer() {
 			if (!waitingForInstallation && !waitingForDisplayArguments && !serverStarted) {
 				disposables.iter(d -> d.dispose());
+				setLanguageConfiguration();
```

Comment:
Why delay this? Seemed fine where it was previously.

Reply:
When creating a new Haxe project (Haxe: Initialize Project) and writing `true.sw<tab><Enter>`, there was no reaction in `onEnterRules`. I thought I fixed it by changing the call pos of `setLanguageConfiguration`, but now I realized that everything was broken again and I do not understand the reason why` onEnterRules` does not work outside of the vshaxe project. Do i need some "release mode build" for extension or something?

Reply:
If i comment out `indentNextLinePattern` it works after reloading sample Haxe project window, for example. `onEnterRules` doesn't works outside of vshaxe project folder even after restart with any rules.

Reply:
Are you sure the dev build of the extension is even being used in the case where it doesn't work? Not sure how you installed vshaxe / how you're testing... Did you just clone it into the extensions folder / do you use the extension development host for testing?

Reply:
Okay, this is because codedox extension overwrites `onEnterRules`. I think Gama will come up with something about it.

Reply:
codedox now replicates the overridden onEnterRules (case, default). Codedox 1.3.1.  Eventually we will either merge the rules when https://github.com/microsoft/vscode/issues/2871 is resolved, or move some/all codedox functionality into vshaxe.

Reply:
Thanks!

Reply:
VSCode issue 2871 mentioned above needs upvotes.  https://github.com/microsoft/vscode/issues/2871